pkg/minikube/constants/constants.go

```diff
@@ -65,7 +65,8 @@ const (
 	DefaultVMDriver     = "virtualbox"
 	DefaultStatusFormat = "minikubeVM: {{.MinikubeStatus}}\n" +
 		"localkube: {{.LocalkubeStatus}}\n"
-	DefaultListFormat         = "- {{.AddonName}}: {{.AddonStatus}}\n"
+	DefaultAddonListFormat    = "- {{.AddonName}}: {{.AddonStatus}}\n"
+	DefaultConfigViewFormat   = "- {{.ConfigName}}: {{.ConfigValue}}\n"
```

Comment:
This should be {{.ConfigKey}} {{.ConfigValue}} as you have it above

- E1014 11:34:27.047476  120837 config_view.go:72] Error executing view template: template: view:1:4: executing "view" at <.ConfigName>: can't evaluate field ConfigName in type config.ConfigViewTemplate

https://github.com/kubernetes/minikube/pull/695/files#diff-42345276571c35ff317855ef4335db2aR34